sdk/compute/Microsoft.Azure.Management.Compute/src/Generated/VirtualMachineScaleSetVMExtensionsOperationsExtensions.cs

```diff
@@ -42,7 +42,7 @@ public static partial class VirtualMachineScaleSetVMExtensionsOperationsExtensio
             /// <param name='extensionParameters'>
             /// Parameters supplied to the Create Virtual Machine Extension operation.
             /// </param>
-            public static VirtualMachineExtension CreateOrUpdate(this IVirtualMachineScaleSetVMExtensionsOperations operations, string resourceGroupName, string vmScaleSetName, string instanceId, string vmExtensionName, VirtualMachineExtension extensionParameters)
+            public static VirtualMachineScaleSetVMExtension CreateOrUpdate(this IVirtualMachineScaleSetVMExtensionsOperations operations, string resourceGroupName, string vmScaleSetName, string instanceId, string vmExtensionName, VirtualMachineScaleSetVMExtension extensionParameters)
```

Comment:
Same comments as above

Reply:
This file has been generated by autorest tool using the last published swagger json (i.e. 30-06-2020).

Reply:
This one is covered by the breaking change in the api, no further action is necessary.